### PR TITLE
 Discord sync doc + various improvements

### PR DESCRIPTION
**Pull request by dotCore-off**

### CODE REVIEW

File: docs/configuration/discord-sync.mdx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+:::
+
+Now, **get back to your Cosmo** instance and:
+* go to your **Dashboard**
+* go in **Settings & Other > Settings > Integrations** tab
+* at the bottom of Discord accordion, **enable Discord Sync** and **fill in the three fields** with previously fetched values
+> *It's **pretty explicit** at this point, copy your **Application Client ID in Discord Client ID** field, etc..*
+* **hit the Save Settings** button and you're good
+
+Here's how it must looks like:
```
Comment (LucaTanks) [inline docs/configuration/discord-sync.mdx:53]: Little nitpick but put “here’s what it should look like”

File: docs/configuration/discord-sync.mdx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+* go to your **Dashboard**
+* go in **Settings & Other > Settings > Integrations** tab
+* at the bottom of Discord accordion, **enable Discord Sync** and **fill in the three fields** with previously fetched values
+> *It's **pretty explicit** at this point, copy your **Application Client ID in Discord Client ID** field, etc..*
+* **hit the Save Settings** button and you're good
+
+Here's how it must looks like:  
+![Cosmo Discord configuration](../../static/img/configuration/cosmo_discord.png)
+
+:::success You're done
```
Comment (LucaTanks) [inline docs/configuration/discord-sync.mdx:56]: Should mention the bot wouldn’t come online as this is a common misconception

**Review by LucaTanks — CHANGES_REQUESTED** [review]

File: docs/configuration/discord-sync.mdx (hunk 2 above)
Comment (dotCore-off) [reply docs/configuration/discord-sync.mdx:56]: fixed my man

File: docs/configuration/discord-sync.mdx (hunk 1 above)
Comment (dotCore-off) [reply docs/configuration/discord-sync.mdx]: nitpicked

**Review by LucaTanks — APPROVED** [review]

**Pull request merged**